lighthouse-plugin-publisher-ads/messages/en-US.json

```diff
@@ -42,20 +42,20 @@
       "title": "Latency of first ad request",
       "failureTitle": "Reduce latency of first ad request",
       "description": "This metric measures the elapsed time from the start of page load until the first ad request is made. Delayed ad requests will decrease impressions and viewability, and have a negative impact on ad revenue. [Learn more](https://developers.google.com/publisher-ads-audits/reference/audits/metrics).",
-      "displayValue": "%s s"
+      "displayValue": "{adReqTime, number, seconds} s"
     },
     "ad-request-from-tag-load": {
       "title": "Latency of first ad request, from tag load",
       "failureTitle": "Reduce latency of first ad request (from tag load)",
       "description": "This metric measures the elapsed time from when the Google Publisher Tag loads until the first ad request is made. [Learn more](https://developers.google.com/publisher-ads-audits/reference/audits/metrics).",
-      "displayValue": "%s s"
+      "displayValue": "{adReqTime, number, seconds} s"
     },
     "ad-top-of-viewport": {
       "title": "No ad found at the very top of the viewport",
       "failureTitle": "Move the top ad further down the page",
       "description": "Over 10% of ads are never viewed because users scroll past them before they become viewable. By moving ad slots away from the very top of the viewport, users are more likely to see ads before scrolling away. [Learn more](https://developers.google.com/publisher-ads-audits/reference/audits/ad-top-of-viewport).",
       "displayValue": "",
-      "failureDisplayValue": "A scroll of %s px would hide half of your topmost ad",
+      "failureDisplayValue": "A scroll of {valueInPx, number} px would hide half of your topmost ad",
```

Comment:
I kind of think the space before "px" should also be removed, to be consistent with CSS styling rules. But I don't feel super strongly about that.